service/tablet_allocator.cc

```diff
@@ -2569,7 +2571,8 @@ class load_balancer {
                 read += shard_load.streaming_read_load;
                 write += shard_load.streaming_write_load;
             }
-            lblogger.info("Node {}: rack={} avg_load={} tablets={} shards={} state={} stream_read={} stream_write={}",
+            auto level = (read + write) > 0 ? seastar::log_level::info : seastar::log_level::debug;
+            lblogger.log(level, "Node {}: rack={} avg_load={} tablets={} shards={} state={} stream_read={} stream_write={}",
```

Comment:
Those logs are supposed to be interpreted in context logged earlier. If the "Examining ..." line is always in debug, we lose association between nodes and DCs here, we only log the rack. So we should probably at least log the DC as well.

Reply:
Yes, I'll add the datacenter here.